lib/src/widgets/chat/video/ds_video_message_bubble.widget.dart

```diff
@@ -39,6 +39,9 @@ class DSVideoMessageBubble extends StatefulWidget {
   /// Style for bubble
   final DSMessageBubbleStyle style;
 
+   // reply id message
```

Comment:
![image](https://github.com/takenet/blip-ds-flutter/assets/14128784/0eda02fc-6c64-4f12-9785-29919ecc2f53)

![image](https://github.com/takenet/blip-ds-flutter/assets/14128784/7368324a-f6e2-411d-9add-b6cd80b2f29e)

The button and video position is not correct